src/sas/sasgui/perspectives/fitting/basepage.py

```diff
@@ -1246,12 +1244,11 @@ def reset_page_helper(self, state):
 
         # we have two more options for smearing
         if self.pinhole_smearer.GetValue():
-            self.dx_min = state.dx_min
-            self.dx_max = state.dx_max
-            if self.dx_min is not None:
-                self.smear_pinhole_min.SetValue(str(self.dx_min))
-            if self.dx_max is not None:
-                self.smear_pinhole_max.SetValue(str(self.dx_max))
+            self.dx_percent = state.dx_percent
+            if self.dx_percent is not None:
+                if state.dx_old:
+                    self.dx_percent = self.data.x[0] / self.dx_percent
```

Comment:
When a fit state is loaded (from a project or analysis file) the data set is loaded separately from the fit parameters and are only brought together again in basepage.py. I plan on cleaning this up for 4.2, but will need to make another large change to make this happen.

dx_old is a new flag to capture when dx_min and dx_max are found in older save states to let basepage know to convert the data from raw values to percent. Once I fully separate the computational and loading code from the gui, This will no longer be needed and get removed.